src/lang/en.json

```diff
@@ -423,6 +423,8 @@
     "telegramProtectContentDescription": "If enabled, the bot messages in Telegram will be protected from forwarding and saving.",
     "supportTelegramChatID": "Support Direct Chat / Group / Channel's Chat ID",
     "wayToGetTelegramChatID": "You can get your chat ID by sending a message to the bot and going to this URL to view the chat_id:",
+    "telegramServerUrl": "(Optional) Server Url",
+    "telegramServerUrlDescription": "Telegram custom bot api server url. Defaults to https://api.telegram.org if empty.",
```

Comment:
Please go into more detail why one would want to change this (custom bots + custom proxys to evade govermental blocking) and link to the docs where appropriate

See the `i18n-t` component to see how to add links to translations ^^